src/data/homeSections.json

```diff
@@ -71,6 +71,22 @@
                     "colorClasses": "bg-green-500 text-white"
                 },
                 "color": "from-amber-500 to-amber-700"
+            },
+            {
+              "title": "PC and Console",
+              "image": "/img/pc-console.png",
+              "link": "/pc-and-console",
+              "alt": "pc-console",
+              "description": "Run Helpshift smoothly with Web apps/games using Webview or QR Code Integration",
```

Comment:
Run Helpshift smoothly with PC and Console apps/games using Webview or QR Code Integration